packages/skygear-core/src/url.test.ts

```diff
@@ -1,9 +1,4 @@
-import {
-  encodeQueryComponent,
-  encodeQuery,
-  decodeQueryComponent,
-  decodeQuery,
-} from "./url";
+import { encodeQueryComponent } from "./url";
```

Comment:
I suppose we can replace the logic in image processing URL builder, so this function can be deleted as well.

Reply:
Updated! So great that we have test in image processing!XD @louischan-oursky